### PR TITLE
PP-826: Do not display vnode_pool in pbsnodes -av output on Windows

### PR DESCRIPTION
**Pull request by lisa-altair**
#### Issue-ID

* **[PP-826](https://pbspro.atlassian.net/browse/PP-826)**

#### Problem description
* On Windows pbsnodes -a output was incorrectly displaying an attribute that should not have been displayed, vnode_pool.

#### Cause / Analysis
* vnode_pool was being initialized and set in initialize_pbsnode()

#### Solution description
* Removed the initialization and set of vnode_pool in initialize_pbsnode()

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [X] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [X] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [X] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: src/server/node_func.c [hunk 1]
```diff
@@ -524,8 +524,8 @@ initialize_pbsnode(struct pbsnode *pnode, char *pname, int ntype)
 	pnode->nd_attr[(int)ND_ATR_Sharing].at_flags =
 		ATR_VFLAG_SET|ATR_VFLAG_DEFLT;
 
-	pnode->nd_attr[(int)ND_ATR_vnode_pool].at_val.at_long = 0;
-	pnode->nd_attr[(int)ND_ATR_vnode_pool].at_flags =
+//	pnode->nd_attr[(int)ND_ATR_vnode_pool].at_val.at_long = 0;
+//	pnode->nd_attr[(int)ND_ATR_vnode_pool].at_flags =
 		ATR_VFLAG_SET|ATR_VFLAG_DEFLT;
```
Comment (latha-subramanian) [inline src/server/node_func.c]: Shouldn't this line also be commented ?

Comment (bhroam) [reply src/server/node_func.c]: Code shouldn't be commented out.  It should either be uncommented or removed.

**Comment by lisa-altair** [comment]
Please hold off on reviews for now.  There are a few more changes I'd like to do.  Thanks.

**Comment by lisa-altair** [comment]
I have made the changes, and tested them.  Please review the code.

**Comment by lisa-altair** [comment]
Thanks @bhroam for catching that.  I have updated the code.

File: src/server/req_manager.c [hunk 2]
```diff
@@ -3649,7 +3649,8 @@ struct batch_request *preq;
 
 	mymom   = pnode->nd_moms[0];
 	vn_pool = pnode->nd_attr[(int)ND_ATR_vnode_pool].at_val.at_long;
-	if (vn_pool > 0) {
+	if ((pnode->nd_attr[(int)ND_ATR_vnode_pool].at_flags & ATR_VFLAG_SET) && 
+	   (vn_pool > 0)) {
```
Comment (bhroam) [inline src/server/req_manager.c]: Here's a suggestion.  Take it as you may.  I'd remove the vn_pool variable.  It looks a little strange to look at pnode on one side and vn_pool on the other.  The vn_pool variable is only used twice.  It'd also make this instance is like the others you changed.  Up to you.

Comment (lisa-altair) [reply src/server/req_manager.c]: I see what you're saying.  However, since it is functionally the same, I will leave it as is.

**Comment by lisa-altair** [comment]
@bhroam if you are okay with me not taking your suggestion, please let me know when I can rebase and squash.

**Comment by bhroam** [comment]
I merged in the PR

**Comment by lisa-altair** [comment]
Thanks @vstumpf and @bhroam!

**Pull request merged**